### ISSUE TITLE
Command-line --port is ignored

### ISSUE THREAD

**Issue by dubek** [EclecticIQ/cabby#35]
I'm running an OpenTAXII server on port 9000, and try to use cabby to access it.  Running with `--port` the value is ignored:

```
$ taxii-push --host 127.0.0.1 --port 9000 --path /service/inbox-a --dest mycollection -f content.txt
2016-07-21 03:39:43,430 INFO: Sending Inbox_Message to http://127.0.0.1/service/inbox-a
2016-07-21 03:39:43,434 INFO: Starting new HTTP connection (1): 127.0.0.1
2016-07-21 03:39:43,435 ERROR: HTTPConnectionPool(host='127.0.0.1', port=80): Max retries exceeded with url: /service/inbox-a (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f983e592198>: Failed to establish a new connection: [Errno 111] Connection refused',))
```

You can see in the first log line that an attempt is made to port 80 and not port 9000.

This works OK if I add a full URL in the path as follows:

    taxii-push --path http://127.0.0.1:9000/services/inbox-a --dest mycollection -f content.txt

Is that the intended usage?
Is the `--port` option actually used?

 
* cabby 0.1.12
* Python 3.4.3

**Comment by traut** [comment]
thanks for the report, I'm looking into it

**Comment by pea-are** [comment]
Same issue appears to occur when performing the create_client command using cabby 0.1.12

`client = create_client(host='127.0.0.1', port=9000, discovery_path=config.get('taxii', 'discovery_path'))`

> Traceback (most recent call last):
  File "C:\Python35\lib\site-packages\requests\packages\urllib3\connection.py", line 142, in _new_conn
    (self.host, self.port), self.timeout, **extra_kw)
  File "C:\Python35\lib\site-packages\requests\packages\urllib3\util\connection.py", line 98, in create_connection
    raise err
  File "C:\Python35\lib\site-packages\requests\packages\urllib3\util\connection.py", line 88, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it

> During handling of the above exception, another exception occurred:

>Traceback (most recent call last):
  File "C:\Python35\lib\site-packages\requests\packages\urllib3\connectionpool.py", line 595, in urlopen
    chunked=chunked)
  File "C:\Python35\lib\site-packages\requests\packages\urllib3\connectionpool.py", line 363, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "C:\Python35\lib\http\client.py", line 1106, in request
    self._send_request(method, url, body, headers)
  File "C:\Python35\lib\http\client.py", line 1151, in _send_request
    self.endheaders(body)
  File "C:\Python35\lib\http\client.py", line 1102, in endheaders
    self._send_output(message_body)
  File "C:\Python35\lib\http\client.py", line 934, in _send_output
    self.send(msg)
  File "C:\Python35\lib\http\client.py", line 877, in send
    self.connect()
  File "C:\Python35\lib\site-packages\requests\packages\urllib3\connection.py", line 167, in connect
    conn = self._new_conn()
  File "C:\Python35\lib\site-packages\requests\packages\urllib3\connection.py", line 151, in _new_conn
    self, "Failed to establish a new connection: %s" % e)
requests.packages.urllib3.exceptions.NewConnectionError: <requests.packages.urllib3.connection.HTTPConnection object at 0x0000000004D98550>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it

**Comment by aukjan** [comment]
Thanks for letting us know @pea-are. We'll look into it!

**Comment by gleblanc1783** [comment]
+1 to this. I was having an issue that I thought was my network. Ran a TCPDump and found out that the port option is not used.

**Comment by traut** [comment]
I'm closing this issue since it should be fixed in the last 0.1.13 version, released today.

Sorry for the delay, guys
